crates/factor-outbound-pg/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "spin-factor-outbound-pg"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+anyhow = "1.0"
+native-tls = "0.2.11"
+postgres-native-tls = "0.5.0"
+spin-core = { path = "../core" }
+spin-factor-outbound-networking = { path = "../factor-outbound-networking" }
+spin-factor-variables = { path = "../factor-variables" }
```

Comment:
I'm not sure we need variables at all (see my other comment), but if we do, and they're only be used in tests, can we move this to `dev-dependencies`?